v1.0.0-rc.5 Release Notes (#4298)
spiceai/spiceai: docs/release_notes/v1.0.0-rc.5.md

# Spice v1.0-rc.5 (Jan 13, 2025)

> Spice v1.0.0-rc.5 is the fifth release candidate for the first major version of Spice.ai OSS. This release focuses production readiness and critical bug fixes. In addition, a new DynamoDB data connector has been and automatic detection for GPU acceleration when running Spice using the CLI.

Reviewer: ```suggestion
Spice v1.0.0-rc.5 is the fifth release candidate for the first major version of Spice.ai OSS. This release focuses production readiness and critical bug fixes. In addition, a new DynamoDB data connector has been added and automatic detection for GPU acceleration when running Spice using the CLI.
```

## Highlights in v1.0-rc.5

- **Automatic GPU Acceleration Detection:** Automatically detect and utilize GPU acceleration when running by CLI. Install AI components locally using the CLI command `spice install ai`. Currently supports NVIdia CUDA and Apple Metal (M-series).

> - **DynamoDB Data Connector:** New DynamoDB data connector.

Reviewer: ```suggestion
- **DynamoDB Data Connector:** Support for querying AWS DynamoDB tables with the DynamoDB Data Connector.

```yaml
datasets:
  - from: dynamodb:users
    name: users
    params:
      dynamodb_aws_region: us-west-2
      dynamodb_aws_access_key_id: ${secrets:aws_access_key_id}
      dynamodb_aws_secret_access_key: ${secrets:aws_secret_access_key}
    acceleration:
      enabled: true
```
```

# Spice v1.0-rc.5 (Jan 13, 2025)

Spice v1.0.0-rc.5 is the fifth release candidate for the first major version of Spice.ai OSS. This release focuses production readiness and critical bug fixes. In addition, a new DynamoDB data connector has been added and automatic detection for GPU acceleration when running Spice using the CLI.

## Highlights in v1.0-rc.5

- **Automatic GPU Acceleration Detection:** Automatically detect and utilize GPU acceleration when running by CLI. Install AI components locally using the CLI command `spice install ai`. Currently supports NVIdia CUDA and Apple Metal (M-series).

> - **DynamoDB Data Connector:** Support for querying AWS DynamoDB tables with the DynamoDB Data Connector.

Reviewer 2: ```suggestion
- **DynamoDB Data Connector:** Query AWS DynamoDB tables using SQL with the new DynamoDB Data Connector.
```

# Spice v1.0-rc.5 (Jan 13, 2025)

> Spice v1.0.0-rc.5 is the fifth release candidate for the first major version of Spice.ai OSS. This release focuses production readiness and critical bug fixes. In addition, a new DynamoDB data connector has been added and automatic detection for GPU acceleration when running Spice using the CLI.

Reviewer 2: ```suggestion
Spice v1.0.0-rc.5 is the fifth release candidate for the first major version of Spice.ai OSS. This release focuses production readiness and critical bug fixes. In addition, a new DynamoDB data connector has been added along with automatic detection for GPU acceleration when running Spice using the CLI.
```

+---------------------------------+----------------------------------------------------------------------------------------------------------------------------------------+
| 71.81.101.73                    | Opera/8.78.(X11; Linux x86_64; ss-ZA) Presto/2.9.186 Version/11.00                                                                     |
| 44.97.163.126                   | Mozilla/5.0 (Windows CE) AppleWebKit/536.1 (KHTML, like Gecko) Chrome/35.0.878.0 Safari/536.1                                          |
| 121.32.180.61                   | Mozilla/5.0 (Linux; Android 6.0) AppleWebKit/534.0 (KHTML, like Gecko) Chrome/23.0.866.0 Safari/534.0                                  |
| 129.101.5.240                   | Mozilla/5.0 (compatible; MSIE 5.0; Windows NT 5.0; Trident/5.1)                                                                        |
| 157.148.178.48                  | Mozilla/5.0 (Macintosh; U; Intel Mac OS X 10_10_0 rv:3.0; bem-ZM) AppleWebKit/532.48.1 (KHTML, like Gecko) Version/5.0 Safari/532.48.1 |

> +---------------------------------+----------------------------------------------------------------------------------------------------------------------------------------+

Reviewer 2: @ewgenius is there some output that's more concise that formats better?

Author: Updated to truncated `des
![CleanShot 2025-01-14 at 11 07 53@2x](https://github.com/user-attachments/assets/09bc9998-c057-4db0-bfb8-56eda21f37df)
cribe users` output
